.github/workflows/ci.yml

```
@@ -177,13 +195,139 @@ jobs:
+                    issue_number: context.issue.number,
+                    body: comment
+                  });
+                }
+              }
+            } catch (error) {
+              console.error('Error processing test results:', error);
+            }
+
+      - name: Deploy to GitHub Pages
+        if: always()
+        uses: peaceiris/actions-gh-pages@v3
```

actionlint (1.7.7)

the runner of "peaceiris/actions-gh-pages@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)